add notes for PKS k8s cluster installations (#58)
chaos-mesh/website: docs/user_guides/installation.md

     helm install chaos-mesh chaos-mesh/chaos-mesh --namespace=chaos-testing
     ```

     > **Note:**
     >

>      > Sometimes we might need to change the default values based on Kubernetes editions.
>      > Example, Docker socket path is different in PKS Kubernetes clusters (VMware K8s Enterprise editions).

Reviewer: ```suggestion
     >Depending on your Kubernetes edition, you may have to change the default values.
     > For example, the Docker socket path is different in PKS Kubernetes clusters (VMware K8s Enterprise edition).
```

Author: @dcalvin  changes are done based on feedback.
